src/main/webapp/src/app/app.module.ts

```diff
@@ -87,15 +87,18 @@ export function initGapi(gapiSession: GapiSession) {
         ],
       } as SocialAuthServiceConfig,
     },
+    /*
```

Comment:
Why are these commented out? Are they intended to be restored at some point, or can they be removed entirely?